### PR TITLE
Add table gcp_sql_database_instance_metric_connections_hourly closes #265

### LINKED ISSUE #265: Add table gcp_sql_database_instance_connections_hourly
**References**
Add any related links that will help us understand the resource, including vendor documentation, related GitHub issues, and Go SDK documentation.

- Add table which will show information about database connection in 5 minute intervals

### PR DESCRIPTION
**Pull request by ParthaI**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>

### Basic info
  
```sql
select
  instance_id,
  timestamp,
  minimum,
  maximum,
  average,
  sample_count
from
  gcp_sql_database_instance_connections_hourly
order by
  timestamp;
```

```
+-----------------------+----------------------+---------+---------+-------------------+--------------+
| instance_id           | timestamp            | minimum | maximum | average           | sample_count |
+-----------------------+----------------------+---------+---------+-------------------+--------------+
| parker-aaa:db-connect | 2021-06-28T11:24:00Z | 8       | 9       | 8.976744186046512 | 43           |
| parker-aaa:db-connect | 2021-06-28T12:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T13:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T14:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T15:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T16:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T17:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T18:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T19:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T20:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T21:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T22:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-28T23:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-29T00:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-29T01:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-29T02:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-29T03:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-29T04:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-29T05:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-29T06:24:00Z | 9       | 9       | 9                 | 60           |
| parker-aaa:db-connect | 2021-06-29T07:24:00Z | 9       | 9       | 9                 | 60           |
+-----------------------+----------------------+---------+---------+-------------------+--------------+
```

### Intervals averaging fewer than 9 connections

```sql
select
  instance_id,
  timestamp,
  round(minimum::numeric,2) as min_connection,
  round(maximum::numeric,2) as max_connection,
  round(average::numeric,2) as avg_connection,
  sample_count
from
  gcp_sql_database_instance_connections_hourly
where average < 9
order by timestamp;
```

```
+-----------------------+----------------------+----------------+----------------+----------------+--------------+
| instance_id           | timestamp            | min_connection | max_connection | avg_connection | sample_count |
+-----------------------+----------------------+----------------+----------------+----------------+--------------+
| parker-aaa:db-connect | 2021-06-28T11:28:00Z | 8.00           | 9.00           | 8.98           | 47           |
+-----------------------+----------------------+----------------+----------------+----------------+--------------+
```

</details>

### CODE REVIEW

**Review by Subhajit97 — APPROVED** [review]

File: docs/tables/gcp_sql_database_instance_metric_connections_hourly.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Table: gcp_sql_database_instance_metric_connections_hourly
+
```
Comment (rajmohanty17) [inline docs/tables/gcp_sql_database_instance_metric_connections_hourly.md:2]: should it be gcp_sql_database_instance_connections_hourly

**Review by rajmohanty17 — CHANGES_REQUESTED** [review]
Validate the table names in the doc.

File: docs/tables/gcp_sql_database_instance_metric_connections_hourly.md (hunk 1 above)
Comment (Subhajit97) [reply docs/tables/gcp_sql_database_instance_metric_connections_hourly.md:2]: @rajmohanty17 We have followed same naming pattern defined in AWS

**Review by rajmohanty17 — APPROVED** [review]
LGTM

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**